src/board.py

```diff
@@ -1,4 +1,4 @@
-from pieces import Pawn, Bishop, Knight, Rook, Queen, King
+from src.pieces import Piece, Pawn, Bishop, Knight, Rook, Queen, King
```

Comment:
I think PEP8 prefers `src.pieces` over `.pieces` with both options being valid. I find the second less verbose thus I would prefer it personally.